src/tools/useLocalStorage.ts

```diff
@@ -18,19 +19,21 @@ export default function useLocalStorage(
   initialValue: string = ""
 ): UseLocalStorage {
   const [value, setValue] = useState(
-    () => window.localStorage.getItem(key) || initialValue
+    () => (isWindowDefined && window.localStorage.getItem(key)) || initialValue
   );
 
   const setItem = (newValue: string) => {
     setValue(newValue);
-    window.localStorage.setItem(key, newValue);
+    isWindowDefined && window.localStorage.setItem(key, newValue);
```

Comment:
Let's use if statements for calling functions, prefer this:
```
if (condition) { doSmth() }
```
over
```
condition && doSmth()
```
I think there's a linter option for that, but forgot how the rule is called

Reply:
Fixed, together with linter (actually, linter was not working at all in the repository, but fixed now)